plugins/csv-import/src/csv.ts

```diff
@@ -207,6 +207,7 @@ function getFieldDataEntryInputForField(
 
         case "divider":
         case "unsupported":
+        default:
```

Comment:
Please, add the missing case explicitly instead, so that future additions highlight this again.

Reply:
<img width="803" height="250" alt="image" src="https://github.com/user-attachments/assets/539d239e-61db-4dad-9a6b-b57c5c65a20c" />